dscreator/datasets/timeseries/sios.py

```diff
@@ -106,6 +106,16 @@ def map_to_cfarray(self, timeseries: NamedTimeseries) -> xr.DataArray:
                         units="S/m",
                     ),
                 )
+            case "fDOMCalib":
+                array = dataarraybytime(
+                    data=timeseries.values,
+                    name="conductivity",
+                    attrs=CFVariableAttrs(
+                        standard_name="concentration_of_colored_dissolved_organic_matter_in_sea_water_expressed_as_equivalent_mass_fraction_of_quinine_sulfate_dihydrate",
```

Comment:
This standard name should be double-checked @knl88

Reply:
Shall we still include the conductivity in the dataset? Sabine did not mention conductivity. She did say oxygen saturation but that this variable in particular should be QCed due to biofouling so either we omit it or we include it but then definitely need to add processing_level for the dataset of NO QC or something similar (we should add that anyway).

Reply:
We can update the dataset with additional parameters later, would be nice to just get it setup to start with.  For processing level we should try to use one of mets keywords [here](https://vocab.met.no/mmd/en/page/?uri=https%3A%2F%2Fvocab.met.no%2Fmmd%2FOperational_Status%2FExperimental) , perhaps we start with experimental. 

Perhaps we can remove conductivity to start with, it is defined [here](https://github.com/NIVANorge/datasetcreator/blob/45bc3a19b617cc78af69e39776da6dd898e66a2b/dscreator/main.py#L40) and you need to swap the other variables with the calibrated once:)

Reply:
Variables updated for calibrated values, we are keeping conductivity and adding fDOM (calibrated) for now as well after discussion with Sabine who gave a green light for 2023-24 data with disclaimer that the data are not completely QCed

Reply:
I am also adding a comment comment = "This dataset has not been completely quality controlled"